Source/Core/FeatureDetection.js

```diff
@@ -199,6 +203,37 @@ define([
         return supportsImageRenderingPixelated() ? imageRenderingValueResult : undefined;
     }
 
+    var supportsWebpPromise;
+    function supportsWebp() {
+        // From https://developers.google.com/speed/webp/faq#how_can_i_detect_browser_support_for_webp
+        if (defined(supportsWebpPromise)) {
+            return supportsWebpPromise;
+        }
+
+        var webpDataUri = 'UklGRiIAAABXRUJQVlA4IBYAAAAwAQCdASoBAAEADsD+JaQAA3AAAAAA';
```

Comment:
Not sure why this is in a variable, just add it to the existing string in the `image.src` line below.